iphone/Maps/UI/BottomMenu/Menu/Cells/BottomMenuItemCell.swift

```diff
@@ -33,28 +32,12 @@ class BottomMenuItemCell: UITableViewCell {
     icon.setStyleAndApply(isEnabled ? "MWMBlack" : "MWMGray")
     label.setStyleAndApply(isEnabled ? "blackPrimaryText" : "blackHintText")
   }
-  
-  func configure(imageName: String, title: String) {
-    icon.image = UIImage.init(named: imageName)
-    label.text = title
-    icon.setStyleAndApply("MWMBlue")
-    label.setStyleAndApply("linkBlueText")
-    badgeBackground.isHidden = true
-    badgeSpacingConstraint.constant = 0
-    badgeBackgroundWidthConstraint.constant = 0
-    isEnabled = true
-    isPromo = true
-  }
-  
+
   override func setHighlighted(_ highlighted: Bool, animated: Bool) {
     guard isEnabled else {
       return
     }
     super.setHighlighted(highlighted, animated: animated)
-    if isPromo {
-      label.setStyleAndApply(highlighted ? "linkBlueHighlightedText" : "linkBlueText")
```

Comment:
nit: Interesting, in this way MM highlighted some new important menu items, right? Maybe we need to do something similar with the recording? At least until it is pressed for the first time by a user?

Reply:
It will looks like:
![Uploading image.png…]()

Reply:
Thanks! Maybe it will be a good way to highlight new features to users... For now let's postpone highlighting, we can do it later if necessary.